app/javascript/components/widget-bar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../lib/theme';
+
+export const CurrentWidgetBar = ({ time }) => (
+  <svg width="568" height="4">
+    <rect width={568 * (time / 20000)} height="4" fill={colors.white} />
```

Comment:
If we replace the hardcoded `20000` here by importing the `const`, we can ensure any changes to the `time` value are reflected everywhere.